TrackPropagation/Geant4e/test/simpleTestPropagator_cfg.py

```diff
@@ -0,0 +1,107 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Era_Run3_cff import Run3
+
+#process = cms.Process("PROPAGATORTEST")
+process = cms.Process("PROPAGATORTEST",Run3)
+
+
+
+  #####################################################################
+  # Message Logger ####################################################
+  #
+process.load("FWCore.MessageService.MessageLogger_cfi")
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('Configuration.StandardSequences.GeometryDB_cff')
+process.load("Configuration.EventContent.EventContent_cff")
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+process.load('Configuration.StandardSequences.MagneticField_38T_cff')
+process.load('Configuration.StandardSequences.EndOfProcess_cff')
+process.load("Configuration.StandardSequences.FrontierConditions_GlobalTag_cff")
+
+from Configuration.AlCa.GlobalTag import GlobalTag
+#process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run1_mc', '')
+process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase1_2022_realistic', '')
+
+process.load("FWCore.MessageService.MessageLogger_cfi")
+
+# MessageLogger customizations
+process.MessageLogger.cerr.enable = False
+process.MessageLogger.cout.enable = True
+labels = ["propTest", "geopro"] # Python module's label
+messageLogger = dict()
+for category in labels:
+      main_key = '%sMessageLogger'%(category)
+      category_key = 'Geant4e' # C++ EDProducer type
+      messageLogger[main_key] = dict(
+              filename = '%s_%s.log' % ("debugG4e", category),
+              threshold = 'DEBUG',
+              default = dict(limit=0)
+              )
+      messageLogger[main_key][category_key] = dict(limit=-1)
+      # First create defaults
+      setattr(process.MessageLogger.files, category, dict())
+      # Then modify them
+      setattr(process.MessageLogger.files, category, messageLogger[main_key])
+
+process.source = cms.Source("EmptySource")
+process.maxEvents = cms.untracked.PSet(
+    input = cms.untracked.int32(1)
+)
+
+## Set up geometry
+process.geopro = cms.EDProducer("GeometryProducer",
+     GeoFromDD4hep= cms.bool(True),
+#     GeoFromDD4hep= cms.bool(False),
+     UseMagneticField = cms.bool(True),
+     UseSensitiveDetectors = cms.bool(False),
+     MagneticField = cms.PSet(
+        UseLocalMagFieldManager = cms.bool(False),
+        Verbosity = cms.bool(False),
+        ConfGlobalMFM = cms.PSet(
+            Volume = cms.string('OCMS'),
+            OCMS = cms.PSet(
+                Stepper = cms.string('G4TDormandPrince45'),
```

Comment:
same as above

Reply:
make a proposal. None of those packages are under my responsibility.
I just made what existed working because I had the need.

Reply:
@civanch please make a proposal

Reply:
Is this taken from here:

https://github.com/cms-sw/cmssw/blob/c28a8e117eb8d1af3ce6d8e0c0035fd68e535d5e/SimG4Core/Application/python/g4SimHits_cfi.py#L129

?

Reply:
YES!

Reply:
followed a suggestion of your

Reply:
the rest trial and errors till did not produce evident error messages

Reply:
Ok, then I would propose use a clone of the parametrization from g4SimHits, so that it stays in synch "automatically"

Reply:
How one clones a PSset from another module? (that we do not what necessarely to import )

Reply:
For single parameters I used it here: 

https://github.com/cms-sw/cmssw/blob/master/CalibTracker/SiPixelLorentzAngle/python/SiPixelLorentzAnglePCLHarvester_cfi.py#L6-L7

I don't know if it works also for entire PSets.
Maybe @makortel can chime in here.

Reply:
PSets can be cloned along `newPSet = existingPSet.clone()`, but I suppose @VinInn would like to avoid importing `SimG4Core.Application.g4SimHits_cfi`. There isn't really a way to avoid importing the PSet from somewhere.

For a test cfg file I don't see a big problem in importing from `SimG4Core.Application`. An alternative would be to abstract the common parts of the `GeometryProducer` and `OscarMTProducer` into a new cfi fragment placed into a package that  `SimG4Core/GeometryProducer` and `SimG4Core/Application` (possibly `TrackPropagation/Geant4e`) can depend on.